kodilla-hibernate/src/main/java/com/kodilla/hibernate/task/Task.java

```diff
@@ -0,0 +1,71 @@
+package com.kodilla.hibernate.task;
+
+import javax.persistence.*;
```

Comment:
Sugeruję, żeby unikać importu z * z kilku powodów. Da się IntelliJ przestawić aby tego nie robił...

Reply:
Przestawiłem IJ w ustawieniach **Class count to use import with ‘*’ and Names count to use static import with ‘*’** trzeba wpisać dużą liczbę lub ręcznie jak się pojawi import z asterisk trzeba użyć skrótu Alt+Enter i wybrać **Replace with single class imports**. Poprawione